src/main/java/net/javadiscord/javabot/systems/qotw/QOTWJob.java

```diff
@@ -39,11 +41,14 @@ protected void execute(JobExecutionContext context, JDA jda) throws JobExecution
 				QuestionQueueRepository repo = new QuestionQueueRepository(c);
 				var nextQuestion = repo.getNextQuestion(guild.getIdLong());
 				if (nextQuestion.isEmpty()) {
-					GuildUtils.getLogChannel(guild).sendMessageFormat("Warning! %s No available next question for QOTW!", config.getQotw().getQOTWReviewRole().getAsMention()).queue();
+					new GuildNotificationService(guild).sendLogChannelNotification("Warning! %s No available next question for QOTW!", config.getQotw().getQOTWReviewRole().getAsMention());
 				} else {
 					QOTWQuestion question = nextQuestion.get();
 					QOTWConfig qotw = config.getQotw();
 					qotw.getSubmissionChannel().getThreadChannels().forEach(thread -> thread.getManager().setArchived(true).queue());
```

Comment:
This should be changed to `.setLocked(true)` so that users cannot unarchive the thread and post messages after the question is over.